File: UI/StoryBoard/src/app/dtos/sprint-dto/sprint-dto.ts
export class SprintDto {
    id : number = 0;
    name : string = "";
    description : string = "";
    creationDate : Date = new Date;
    initionDate : Date = new Date;
    endDate : Date = new Date;
    idProject: number = 0;

    static createFromFormValues(formValues: any): SprintDto {
        const sprint = new SprintDto();
        sprint.name = formValues.name;
        sprint.description = formValues.description;
        sprint.creationDate = formValues.creationDate;
        sprint.initionDate = formValues.initionDate;
        sprint.endDate = formValues.endDate;
        sprint.idProject = formValues.idProject;
        return sprint;
    }
}

File: WebAPI2/StoryBoard/Dto/SprintDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Dto;

public class SprintDto
{
    public int Id { get; set; }
    public string Name { get; set; }
    public string Description { get; set; }
    public DateTime CreationDate { get; set; }
    public DateTime InitionDate { get; set; }
    public DateTime EndDate { get; set; }
    public int IdProject { get; set; }
}
